utils/vast_parser.js

```diff
@@ -123,14 +123,14 @@ var VastParser = function() {
     var nonLinearAds = jqueryXML.find("NonLinearAds");
 
     if (result.type === AD_TYPE.WRAPPER) { result.VASTAdTagURI = jqueryXML.find("VASTAdTagURI").text(); }
-    result.error = filterEmpty(jqueryXML.find("Error").map(function() { return $(this).text(); }));
-    result.impression = filterEmpty(jqueryXML.find("Impression").map(function() { return $(this).text(); }));
-    result.title = _.first(filterEmpty(jqueryXML.find("AdTitle").map(function() { return $(this).text(); })));
+    result.error = filterEmpty(jqueryXML.find("Error").map(function() { return OO.$(this).text(); }));
```

Comment:
I know you aren't adding this line but can you fix the spacing fo this  to agree with our coding guidelines? The return shouldn't be in the same line as the rest of the code, so we can put breakpoints in there to debug if necessary.

Something more like this:
result.error = filterEmpty(jqueryXML.find("Error").map(function() { 
  return OO.$(this).text(); 
}));

And do that for the other two lines below this.

Reply:
done